Makefile

```diff
@@ -96,6 +96,11 @@ test:
 test-examples:
 	python -m pytest -n auto --dist=loadfile -s -v ./examples/pytorch/
 
+# Run benchmark
+
+benchmark:
+	python3 benchmark/benchmark.py --config-dir benchmark/config --config-name generation --commit=diff backend.model=google/gemma-2b backend.cache_implementation=null,static backend.torch_compile=false,true --multirun
```

Comment:
can this install the lib as well?

Reply:
So far this Makefile doesn't install stuff in the commands. It's the users' to install beforehand.
Putting an installation here will (try to) install in each run which is not ideal.

I will add a comment.